lib/text_diff.mli

```diff
@@ -1,7 +1,9 @@
-(** Conversion from library diff abstraction to textual diff, line based
-    diff *)
+(** Utilities for custom diff printing  *)
 
-type t = Diffutils.Diff.t String_map.t
+type conflict2 = { orig : string list; new_ : string list }
+type diff = Diff of conflict2
```

Comment:
Since there is a single variant here, we can remove this type and turn our type `t` into:
```
type t = conflict2 list String_map.t
```